lib/ui/common/carousel_view.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/widgets.dart';
+
+class _DefaultConstants {
+  static const int _initialPage = 0;
+  static const double _viewPortFraction = 0.8;
+}
+
+class CarouselView extends StatelessWidget {
```

Comment:
I think that we can use PageView directly

Reply:
I agree, unless we see some grouping display logic for the stages

Reply:
It's just to add the viewPortFraction value on the PageController for the PageView with a default value. I leave it for now, we always can use the PageView. It will works too